python/lancedb/query.py

```diff
@@ -230,6 +231,23 @@ def to_arrow(self) -> pa.Table:
         )
         return self._table._execute_query(query)
 
+    def to(self, model: Type[LanceModel]) -> List[LanceModel]:
```

Comment:
can we rename this to `to_pydantic` , `to` is too wide to be tight with one specific library.

Reply:
done